provisiondomainmanager_policy.tf

```diff
@@ -120,34 +129,16 @@ data "aws_iam_policy_document" "provisiondomainmanager_policy_ecr_ecs_elb_events
     ]
 
     resources = [
-      "arn:aws:ecr::${local.userservices_account_id}:repository/domain-manager-*",
+      "arn:aws:ecr:*:${local.userservices_account_id}:repository/domain-manager-*",
     ]
   }
 
-  statement {
-    actions = [
-      "ecr:DescribeRegistry",
-      "ecr:DescribeRepositories",
-      "ecr:GetAuthorizationToken",
-      "ecr:ListTagsForResource",
-      "ecr:TagResource",
-      "ecr:UntagResource",
-    ]
-
-    resources = ["*"]
-  }
-
   statement {
     actions = [
       "ecs:CreateService",
       "ecs:DeleteCluster",
       "ecs:DeleteService",
```

Comment:
Use tags to restrict actions like this.

Reply:
@jsf9k I'm not sure what you're pointing out here. Are you referring to the ECS actions? What's wrong with limiting by resource in this case?